### DISCUSSION TITLE
Airflow - LDAP Authentication with RBAC

### DISCUSSION THREAD

**Discussion by basanthcse** [apache/airflow#14245] [Q&A]
Airflow Version 1.10.12
Flask-AppBuilder 2.3.4
We have implemented LDAP authentication with RBAC in Airflow user authentication. I have one situation where every user is able to login as Admin, if i mention AUTH_USER_REGISTRATION_ROLE = "Admin" in webserver_config.py file. I have two ldap group (airflow_admin and airflow_dev groups), i need airflow_admin user to login as ADMIN role and airflow_dev group users as OP role. How can I dynamically assign the AUTH_USER_REGISTRATION_ROLE based on the user's LDAP role?

**Comment by lewismc** [comment]
I am also having LDAP authentication issues @basanthcse 
I registered https://github.com/apache/airflow/discussions/14310
Are you upgrading to Airflow 2.X or still working on 1.x?
